README.md

```diff
@@ -51,6 +51,26 @@ databases are required.
   https://dvc.org/doc/user-guide/experiment-management/experiments-overview
 [data versioning]: https://dvc.org/doc/use-cases/versioning-data-and-model-files
 
+## Contents
+
+- [Quick start](#quick-start)
```

Comment:
TOC is too low and too heavy ... can we have something similar to the top menu of the DVC repo? also make sure that anchors work at all in the marketplace and in the VS Code itself when you open the extension's page

Reply:
Yeah good idea. Done, see under https://github.com/iterative/vscode-dvc/tree/readme/known-limitations#why-prefer-this-extension:

![image](https://user-images.githubusercontent.com/1477535/175756661-c341e4dc-951d-47f4-9d78-8d12dba1e67f.png)

Reply:
Thanks @jorgeorpinel !Why not under the badges?

Also, can we check if it works in all the environments?

Reply:
you can test if it works in product by running `yarn build` then installing the build VSIX manually into VS Code:

https://user-images.githubusercontent.com/37993418/175842637-745c3c06-34c2-4cae-94c5-d82a28dbe492.mov

Reply:
Hopefully, if it works in the product it will also work in the marketplace but we'll need to verify this when we publish the next release.

Reply:
> Why not under the badges?

After the value prop and competitive advantages seems a good place for the ToC. Especially because if I add any more links (i.e. "Why this extension") it will probably start wrapping in most screens ():

![image](https://user-images.githubusercontent.com/1477535/176093989-bd0356b0-15f8-41f8-82f3-282f5aa081e7.png)

Reply:
> > check if it works in all the environments
>
> running yarn build then installing the build VSIX manually

Looks OK for me. It's just some links and big dots •

![image](https://user-images.githubusercontent.com/1477535/176094828-eaa8b650-00ee-4022-aa29-4a2d0f763c53.png)

Reply:
Hmm ... contents that far in the text looks very strange to me tbh.

Can just skip "Why prefer .." but still move it to the top? It will fit one line and it should be better I think?